### PR TITLE
replace element to macro

### PR DESCRIPTION
**Pull request by mutoe**
> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'

> What was wrong/why is this fix needed? (quick summary only)

- element reference should using `HTMLElement` macro
- `nbsp` special char should replace to space
- correct some grammar issues

> Anything else that could help us review it

### CODE REVIEW

File: files/en-us/web/accessibility/aria/roles/listbox_role/index.html [hunk 1]
```diff
@@ -121,54 +121,54 @@ <h4 id="Toggling_the_state_of_an_option_in_a_multi_select_listbox">Toggling the
 </ol>
 
 <div class="note">
-<p> <span class="ILfuVd yZ8quc">The first rule of ARIA use is you can use a native feature with the semantics and behavior you require already built in, instead of re-purposing an element and <strong>adding</strong> an ARIA role, state or property to make it accessible, then do so. The &lt;select&gt; element with descendant &lt;option&gt; elements handles all the needed interactions natively.</span></p>
+<p> <span class="ILfuVd yZ8quc">The first rule of ARIA use is you can use a native feature with the semantics and behavior you require already built in, instead of re-purposing an element and <strong>adding</strong> an ARIA role, state or property to make it accessible, then do so. The {{HTMLElement('select')}} element with descendant {{HTMLElement('option')}} elements handles all the needed interactions natively.</span></p>
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/roles/listbox_role/index.html:124]: Can you remove the span class here too?

File: files/en-us/web/accessibility/aria/roles/listbox_role/index.html [hunk 2]
```diff
@@ -121,54 +121,54 @@ <h4 id="Toggling_the_state_of_an_option_in_a_multi_select_listbox">Toggling the
...
 </div>
 
 <h2 id="Examples">Examples</h2>
 
 <h4 class="p1" id="Example_1_A_single_select_listbox_that_uses_aria-activedescendant">Example 1: A single select listbox that uses aria-activedescendant</h4>
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/roles/listbox_role/index.html:129]: Please remove classes p1, p2, … in this page.

File: files/en-us/web/accessibility/aria/roles/listbox_role/index.html [hunk 3]
```diff
@@ -121,54 +121,54 @@ <h4 id="Toggling_the_state_of_an_option_in_a_multi_select_listbox">Toggling the
...
+	<li><a href="https://w3c.github.io/aria-practices/examples/listbox/listbox-collapsible.html">Collapsible Dropdown Listbox Example</a>: Single-select collapsible listbox that expands when activated, similar to HTML {{HTMLElement('select')}} with the attribute <code>size="1"</code>.</li>
 	<li><a href="https://w3c.github.io/aria-practices/examples/listbox/listbox-rearrangeable.html">Example Listboxes with Rearrangeable Options</a>: Examples of both single-select and multi-select listboxes with accompanying toolbars where options can be added, moved, and removed.</li>
 </ul>
 
 <h2 id="Best_practices">Best practices</h2>
 
 <ul class="ul1">
-	<li class="li3">To be keyboard accessible, authors should <a href="https://www.w3.org/TR/wai-aria-1.1/#managingfocus">manage focus</a> of all descendants of this role.</li>
+	<li class="li3">To be keyboarded accessible, authors should <a href="https://www.w3.org/TR/wai-aria-1.1/#managingfocus">manage focus</a> of all descendants of this role.</li>
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/roles/listbox_role/index.html:171]:
```suggestion
	<li class="li3">To be keyboard-accessible, authors should <a href="https://www.w3.org/TR/wai-aria-1.1/#managingfocus">manage focus</a> of all descendants of this role.</li>
```

File: files/en-us/web/accessibility/aria/roles/listbox_role/index.html [hunk 4]
```diff
@@ -121,54 +121,54 @@ <h4 id="Toggling_the_state_of_an_option_in_a_multi_select_listbox">Toggling the
...
+	<li><a href="https://w3c.github.io/aria-practices/examples/listbox/listbox-scrollable.html">Scrollable Listbox Example</a>: Single-select listbox that scrolls to reveal more options, similar to HTML {{HTMLElement('select')}} with <code>size</code> attribute greater than one.</li>
+	<li><a href="https://w3c.github.io/aria-practices/examples/listbox/listbox-collapsible.html">Collapsible Dropdown Listbox Example</a>: Single-select collapsible listbox that expands when activated, similar to HTML {{HTMLElement('select')}} with the attribute <code>size="1"</code>.</li>
 	<li><a href="https://w3c.github.io/aria-practices/examples/listbox/listbox-rearrangeable.html">Example Listboxes with Rearrangeable Options</a>: Examples of both single-select and multi-select listboxes with accompanying toolbars where options can be added, moved, and removed.</li>
 </ul>
 
 <h2 id="Best_practices">Best practices</h2>
 
 <ul class="ul1">
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/roles/listbox_role/index.html:170]: Please remove these strange classes ul1, …

File: files/en-us/web/accessibility/aria/roles/listbox_role/index.html [hunk 5]
```diff
@@ -6,27 +6,27 @@
   - Accessibility
   - NeedsContent
 ---
-<p><span class="seoSummary">The <code>listbox</code> role is used for lists from which a user may select one or more items which are static and, unlike HTML &lt;select&gt; elements, may contain images.</span></p>
+<p><span class="seoSummary">The <code>listbox</code> role is used for lists from which a user may select one or more items which are static and, unlike HTML {{HTMLElement('select')}} elements, may contain images.</span></p>
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/roles/listbox_role/index.html:9]: Please remove the span too, useless if it covers the whole first paragraph.

File: files/en-us/web/accessibility/aria/roles/listbox_role/index.html [hunk 6]
```diff
@@ -6,27 +6,27 @@
...
 
 <p>Elements with the role <code>listbox</code> have an implicit <code>aria-orientation</code> value of <code>vertical</code>.</p>
 
 <p class="p1">When a list is tabbed to, the first item in the list will be selected if nothing else already is. Up/down arrows navigate the list, and pressing Shift + Up/Down arrows will move and extend the selection. Typing one or more letters will navigate the list items (same letter goes to each item starting with that, different letters go to the first item starting with that entire string). If the current item has an associated context menu, Shift+F10 will launch that menu. If list items are checkable, Space can be used to toggle <a href="https://www.w3.org/TR/wai-aria-practices/#checkbox">checkboxes</a>. For selectable list items, Space toggles their selection, Shift+Space can be used to select contiguous items, Ctrl+Arrow moves without selecting, and Ctrl+Space can be used to select non-contiguous items. It is recommended that a checkbox, link or other method be used to select all items, and Ctrl+A could be used as a shortcut key for this.</p>
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/roles/listbox_role/index.html:19]: Please remove the classes p1, …

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Thanks a lot, there are a few other "oddities" to clean on that page, let's take this opportunity to clean it further.

**Comment by mutoe** [comment]
@teoli2003 Thanks for your review. I have removed all useless classes in this file, please check again :)

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:

**Pull request merged**